spec/dataframe_spec.rb

```diff
@@ -2927,6 +2927,19 @@
         )
       )
     end
+
+    it 'works' do
+      categories = %i[jan feb mar apr may jun jul aug sep oct nov dec]
+      df = Daru::DataFrame.rows([
+        [2014, 2, 1600.0, 20.0],
+        [2014, 3, 1680.0, 21.0],
+        [2016, 2, 1600.0, 20.0],
+        [2016, 4, 1520.0, 19.0],
+      ], order: [:year, :month, :visitors, :days])
+      df[:averages] = df[:visitors] / df[:days]
+      df[:month] = df[:month].map{|i| categories[i - 1]}
```

Comment:
I'll rewrite it into something more test-alike :) Or we could lost the same functionality in the future.